.github/workflows/management-github_management.yml

```
@@ -30,9 +30,18 @@ jobs:
+        id: check_config
+        run: |
+          if [ -f .github/config.json ]; then echo "exists=true" >> "$GITHUB_OUTPUT"; else echo "exists=false" >> "$GITHUB_OUTPUT"; fi
+
       - name: Release Mastermind
+        if: steps.check_config.outputs.exists == 'true'
         uses: Resnovas/smartcloud@1.0.0-beta.8 # 1.0.0-beta.8
         with:
           GITHUB_TOKEN: ${{ secrets.BOT_TOKEN || github.token }}
           config: .github/config.json
           configRef: main
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)